tests/continuous_integration_test.py

```diff
@@ -19,110 +19,125 @@
+            "name": "sink"
+        },
+        {
+            "config": {
+            "Reactor": {
+            "assem_size": "1000",
+            "cycle_time": "18",
+            "fuel_incommods": {"val": "freshfuel"},
+            "fuel_inrecipes": {"val": "fresh_uox"},
+            "fuel_outcommods": {"val": "spent_fuel"},
+            "fuel_outrecipes": {"val": "spent_uox"},
+            "n_assem_batch": "1",
+            "n_assem_core": "3",
+            "power_cap": "1000",
+            "refuel_time": "1"
+            }
+            },
+            "name": "reactor"
+        }
+        ],
+        "recipe": [
+        {
+            "basis": "mass",
+            "name": "fresh_uox",
+            "nuclide": [{"comp": "0.711", "id": "U235"}, {"comp": "99.289", "id": "U238"}]
+        },
+        {
+            "basis": "mass",
+            "name": "spent_uox",
+            "nuclide": [{"comp": "50", "id": "Kr85"}, {"comp": "50", "id": "Cs137"}]
+        }
+        ],
+        "region": {
+        "config": {"NullRegion": "\n      "},
+        "institution": {
+            "config": {
+            "TimeSeriesInst": {
+            "calc_method": calc_methods[x],
+            "commodities": {"val": ["POWER_reactor_1000", "freshfuel_source_3000"]},
+            "demand_eq": "1000",
+            "demand_std_dev": "0.0",
+            "record": "1",
+            "steps": "1"
+            }
+            },
+            "name": "source_inst"
+        },
+        "name": "SingleRegion"
+        }
+        }
+        }
+
+total = {}
+
+for x in range(0,len(calc_methods)):
+    name = "test_cont_"+calc_methods[x]
+    input_file = name+".json"
+    output_file = name+".sqlite"
     with open(input_file, 'w') as f:
-        json.dump(INPUT, f)
+        json.dump(test_cont_input[x], f)
     s = subprocess.check_output(['cyclus', '-o', output_file, input_file],
                                 universal_newlines=True, env=ENV)
-    # check if ran successfully
-    assert("Cyclus run successful!" in s)
 
-    # check if timeseriesdemandfreshfuel, timeseriessupplyfreshfuel, timeseriespower, timeseriessupplypower exist 
-    cur = get_cursor('test_cont_int.sqlite')
+    cur = functions.get_cursor(output_file)
     fuel_demand = cur.execute("select time, sum(value) from timeseriesdemandfreshfuel group by time").fetchall()
     fuel_supply = cur.execute("select time, sum(value) from timeseriessupplyfreshfuel group by time").fetchall()
     power_demand = cur.execute("select time, sum(value) from timeseriespower group by time").fetchall()
     power_supply = cur.execute("select time, sum(value) from timeseriessupplypower group by time").fetchall()
-    total = fuel_demand[0][0]+fuel_supply[0][0]+power_demand[0][0]+power_supply[0][0] 
-    assert(total >= 0)
+    # Adding up the first time step of each table  
+    total[calc_methods[x]] = fuel_demand[0][0]+fuel_supply[0][0]+power_demand[0][0]+power_supply[0][0] 
```

Comment:
why are you checking if the first time step of each table? Are you trying to check and see if the table exists?

Reply:
yes essentially